main.go

```diff
@@ -150,19 +153,30 @@ func collectStatsForWithError(proc *process.Process, withError bool) (*MozProces
 
 	statistics := new(MozProcessStat)
 
+	/* Attempting to collect the CPU percentage as well to better query for the
+	instance usage.
+	*/
+	cpu_percent, err := proc.CPUPercent()
+	if err != nil {
+		if withError {
+			log.Printf("CPU Percent: %s\n", err)
+		}
+	}
+
 	cpu, err := proc.Times()
 	if err != nil {
 		if withError {
 			log.Printf("CPU Times: %s\n", err)
 		}
 	} else {
 		// Three significant digits for the cpu times.
-		statistics.CPU = CPUTimesStat{
+		statistics.CPU = ProcCPUStat{
 			math.Round(cpu.User*1000) / 1000,
 			math.Round(cpu.System*1000) / 1000,
 			math.Round(cpu.Idle*1000) / 1000,
 			math.Round(cpu.Iowait*1000) / 1000,
-			math.Round(cpu.Steal*1000) / 1000}
+			math.Round(cpu.Steal*1000) / 1000,
+			cpu_percent}
```

Comment:
For now I'm not round this up as I wanted to see the actual numbers after comma.